webrender/src/tiling.rs

```diff
@@ -1723,6 +1794,26 @@ impl ScreenTile {
                             alpha_task_stack.push(prev_task);
                         }
                     }
+
+                    // Create a task for the layer mask, if needed
```

Comment:
The `clip_cache_info` would be `None` for stuff with just rectangle clips and no images/rounded borders. That's what "if needed" part implies here. I'll specify it in more detail.